src/jira/client/index.ts

```diff
@@ -153,13 +153,22 @@ async function getJiraClient(
 							}
 						}
 					),
-				delete: (gitHubInstallationId: string | number) =>
-					instance.delete(`/rest/devinfo/0.10/bulkByProperties`,
-						{
-							params: {
-								installationId: gitHubInstallationId
-							}
-						})
+				delete: async (gitHubInstallationId: string | number) =>
+					Promise.all([
```

Comment:
this needs to be either returned or awaited.

Reply:
nevermind... side by side view cut off the arrow.

Reply:
it _is_ returned due to the `=>` notation, is it not?